README.md

```diff
@@ -141,6 +141,7 @@ metrics_client = require('metrics')
 
 #### `client_obj.collect()`
   Returns concatenation of `obj:collect()` across all collectors created.
+  See above `counter_obj:collect()` for details.
```

Comment:
Why only `counter_obj`?

Reply:
Because it is the same as `gauge_obj:collect()`.
I think it is better to move `client_obj.collect()` section to the top (after `metrics_client.connect()`), add there `data` description, (and maybe rename `data` -> `observation`), and in `{counter,gauge,hist}:collect()` sections just reference to   `client_obj.collect()`.

Reply:
Good idea, agreed. We need to:
1) introduce `observation` object,
2) outline that is same for both `counter` and `gauge` object and
3) refer to its section in all `:collect()` calls.